_pages/en_US/homebrew-launcher-(browserhax-2020).txt

```diff
@@ -0,0 +1,73 @@
+---
+title: "Homebrew Launcher (BrowserHax 2020)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of accessing the Homebrew Launcher takes advantage of some browser oversights in the native 3DS Internet Browser.
+
+If your device already has access to the Homebrew Launcher (whether through BrowserHax, Pichaxx or another exploit), you can skip to [USM](installing-boot9strap-(hbl-usm)).
+{: .notice--success}
+
+### New, Old and Region
+
+To use this guide you will need to know if your console is "New" or "Old", and what region it is from.
+
+1. Open [this image](https://3ds.eiphax.tech/pic/newold.png) in a new tab or window on your computer.
+  + If your console has Start and Select in the bottom right as circular buttons, it is a New console.
+  + If your console has Start and Select under the touch screen as rectangular buttons, it is an Old console.
+1. Remember which designation (New or Old) your console falls under. You will need to know this later.
+1. Power on your console.
+1. Open System Settings.
+1. Look at the top screen.
+1. In the bottom right of the top screen, you will see a firmware version string, followed by a letter. This letter is your region.
+  + E designates Europe/Oceania.
+  + U designates USA/South America.
+  + J designates Japan.
+  + Any letter NOT listed here is not compatible with this method.
+1. Remember your region designation. You will need to know this later.
+
+### What You Need
+
+* The latest release of [the Homebrew Launcher](https://github.com/fincs/new-hbmenu/releases/latest)
+* If using a "NEW" console: the latest release zip file of [New BrowserHax](https://github.com/zoogie/new-browserhax/releases/latest)
+* If using an "OLD" console: the latest release zip file of [Old BrowserHax](https://github.com/zoogie/old-browserhax/releases/latest)
+* An active WiFi connection on your console
+
+### Instructions
+
+#### Section I - Prep Work
+
+1. Copy `boot.3dsx` from the Homebrew Launcher release to the root of your SD card
+1. Unzip the BrowserHax folder
+1. Inside the BrowserHax folder, open the folder which corresponds to your region (`EUROPE`, `JAPAN` or `USA`)
+1. Copy both of these files to the root of your SD card
+  + if you do not see a file extension, do not add one
+
+#### Section II - Updating the System
+
+1. Update your device by going to System Settings, then "Other Settings", then going all the way to the right and using "System Update"
+  + This is to ensure your browser version is compatible with this exploit
```

Comment:
There's no cart update for 11.13.0, so broken browsers would likely be in the vast minority -- this section may be unnecessary.

Reply:
this was mostly for sanity's sake, i agree that it's unlikely we would have an issue.